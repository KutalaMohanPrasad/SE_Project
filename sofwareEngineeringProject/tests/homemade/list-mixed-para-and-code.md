*   a list containing a block of code with paragraph not surrounding code

	    10 PRINT HELLO INFINITE
	    20 GOTO 10
    
    second paragraph with paragraph not surrounding code

	    10 PRINT HELLO INFINITE
	    20 GOTO 10
    
