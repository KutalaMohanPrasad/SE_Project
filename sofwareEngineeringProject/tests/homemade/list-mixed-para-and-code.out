<ul>
<li><p>a list containing a block of code with paragraph not surrounding code</p>

<pre><code>10 PRINT HELLO INFINITE
20 GOTO 10
</code></pre>

<p>second paragraph with paragraph not surrounding code</p>

<pre><code>10 PRINT HELLO INFINITE
20 GOTO 10
</code></pre></li>
</ul>